smile_id_core/api_base.py

```diff
@@ -39,7 +39,42 @@ def _make_request(
         if method != "get" and data is not None:
             data = json.dumps(data)
 
-        url = url.format(server_url=cls.SERVER_URL)
+        url = url.format(server_url=cls.SERVER_URL, api_version=1)
```

Comment:
Do I understand correctly that this method differs from `_make_request_async` only by the `api_version` ? If so, does it make sense to create another method that would take api_version as parameter and eliminate the duplication of 30 lines of code?

Reply:
yes please! I missed that part when reviewing